### PR TITLE
Prepare release 0.9.0.0

### PR DESCRIPTION
**Pull request by sjakobi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 ## Changelog for the `threepenny-gui` package
 
+**0.9.0.0** – Maintenance and snapshot release
+
+* The events `contextmenu`, `mousedown`, `mousemove` and `mouseup` now return
+  `Double` coordinates instead of `Int`s. This change reflects updates to the
+  underlying browser APIs and the jQuery library. [#238][]
```
Comment (sjakobi) [inline CHANGELOG.md:7]: I'm not super happy with this wording. @HeinrichApfelmus, @brprice, if you have suggestions how to improve it, they'd be most welcome!

File: CONTRIBUTORS [hunk 2]
```diff
@@ -2,6 +2,7 @@ Heinrich Apfelmus <https://github.com/HeinrichApfelmus>
 Daniel Austin     <https://github.com/daniel-r-austin>
 Jeremy Barisch-Rooney <https://github.com/barischj>
 Steve Bigham
+brprice           <https://github.com/brprice>
```
Comment (sjakobi) [inline CONTRIBUTORS:5]: Would you possibly prefer to see a different name used here, @brprice?

Comment (brprice) [reply CONTRIBUTORS:5]:
```suggestion
Ben Price         <https://github.com/brprice>
```

**Comment by sjakobi** [comment]
Thanks @brprice! It's on Hackage now: http://hackage.haskell.org/package/threepenny-gui-0.9.0.0

**Comment by HeinrichApfelmus** [comment]
Thank you again for your help, Simon! Looks great to me. 😄

**Pull request merged**